build.gradle

```diff
@@ -49,6 +49,11 @@ dependencies {
     annotationProcessor group: "io.kestra", name: "processor", version: kestraVersion
     compileOnly group: "io.kestra", name: "core", version: kestraVersion
     compileOnly group: "io.kestra", name: "script", version: kestraVersion
+
+    implementation "com.konghq:unirest-java:3.14.4"
+    implementation "com.vladsch.flexmark:flexmark-html2md-converter:0.64.8"
+    implementation "org.json:json:20240303"
```

Comment:
Can you remove this one and use the standard [JsonArray](https://docs.oracle.com/javaee/7/api/javax/json/JsonArray.html)?